common/src/main/java/com/microsoft/identity/common/internal/fido/LegacyFido2ApiException.java

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+package com.microsoft.identity.common.internal.fido;
+
+import com.microsoft.identity.common.java.exception.BaseException;
+
+/**
+ * An exception class which holds attributes from the legacy FIDO2 API error responses.
+ */
+public class LegacyFido2ApiException extends BaseException {
+
+    public static final String NULL_OBJECT = "null_object";
```

Comment:
I think this constant was already defined somewhere in the code

Reply:
null_object is also defined in ClientException, but LegacyFido2ApiException is different exception class which extends BaseException, so I'm not sure that these two exceptions should share strings.

Reply:
I think there is a file called as `ErrorStrings.java` (or something like that). We should define all common consts in that file

Reply:
Done: 6922a39600e0ade06e818ae2f3dbdf856258b473